src/models/httpRequest.ts

```diff
@@ -11,7 +11,8 @@ export class HttpRequest {
         public headers: RequestHeaders,
         public body: string | Stream | undefined,
         public rawBody: string | undefined,
-        public requestVariableCacheKey?: RequestVariableCacheKey) {
+        public requestVariableCacheKey?: RequestVariableCacheKey,
+        public confirmSendMsg?: string) {

```

Comment:
The `confirmSendMsg` field is only used for request send confirmation, I think we don't need to put it in the request text.

Reply:
And what's about serialization?

Reply:
I think no, we don't need to show this info for history